packages/product-components/src/utils/mapTrezorModelToIcon.ts

```diff
@@ -1,11 +1,24 @@
 import { IconName } from '@trezor/components';
 import { DeviceModelInternal } from '@trezor/connect';
 
-export const mapTrezorModelToIcon: Record<DeviceModelInternal, IconName> = {
+export const mapTrezorModelToIconFilled: Record<DeviceModelInternal, IconName> = {
     [DeviceModelInternal.T1B1]: 'trezorModelOneFilled',
     [DeviceModelInternal.T2T1]: 'trezorModelTFilled',
     [DeviceModelInternal.T2B1]: 'trezorSafe3Filled',
     [DeviceModelInternal.T3B1]: 'trezorSafe3Filled',
     [DeviceModelInternal.T3T1]: 'trezorSafe5Filled',
     [DeviceModelInternal.T3W1]: 'trezorSafe7Filled',
+
+    [DeviceModelInternal.UNKNOWN]: 'trezorModelOneFilled', // Just to provide something that wont break UI
+};
+
+export const mapTrezorModelToIcon: Record<DeviceModelInternal, IconName> = {
```

Comment:
:thought_balloon: maybe there could be notice that these are deprecated. Otherwise you have to click through `IconName` to find out. Perhaps rename to `mapTrezorModelToDeprecatedIcon`